apps/server/src/shared/infra/collaborative-storage/strategy/httpRequests/Nextcloud Groups.http

```diff
@@ -11,13 +11,13 @@ Authorization: Basic capcakes {{CAPCAKESPASS}}
 OCS-APIRequest: true
 Accept: application/json
 ###
-# curl -u capcakes:4MbdOV3LSUOnolGzB2Ab8cL0tt -X GET 'https://nextcloud-nbc.dbildungscloud.dev/apps/groupfolders/folders/1' -H "OCS-APIRequest: true" -H "Accept: Application/json"
+# curl -X GET 'https://nextcloud-nbc.dbildungscloud.dev/apps/groupfolders/folders/1' -H "OCS-APIRequest: true" -H "Accept: Application/json"
```

Comment:
Top but i think we should change the credentials

Reply:
thought they were deleted, anyway yes we should..

Reply:
they are deleted but in the git commit history. you could also let them deleted by github.. or easier way let them change from sre

Reply:
we changed the credentials.